.github/workflows/convergence-tests.yml

```
@@ -18,13 +18,9 @@ on:
         default: "esm2_native_te"
```

actionlint (1.7.8)

default value "esm2_native_te" of "model_config" input is not included in its options "\"esm2_native_te_650m\", \"esm2_native_te_3b\", \"esm2_native_te_15b\""

(events)